cranelift/assembler/meta/src/dsl/format.rs

```diff
@@ -80,12 +111,16 @@ impl core::fmt::Display for Operand {
         let Self { location, mutability, extension } = self;
         write!(f, "{location}")?;
         let has_default_mutability = matches!(mutability, Mutability::Read);
-        let has_default_extension = matches!(extension, Extension::None);
-        match (has_default_mutability, has_default_extension) {
-            (true, true) => {}
-            (true, false) => write!(f, "[{extension}]")?,
-            (false, true) => write!(f, "[{mutability}]")?,
-            (false, false) => write!(f, "[{mutability},{extension}]")?,
+        //let has_default_extension = matches!(extension, Extension::None);
+        //match (has_default_mutability, has_default_extension) {
+        //(true, true) => {}
+        //(true, false) => write!(f, "[{extension}]")?,
+        //(false, true) => write!(f, "[{mutability}]")?,
+        //(false, false) => write!(f, "[{mutability},{extension}]")?,
+        //}
+        match has_default_mutability {
+            true => write!(f, "[{extension}]")?,
+            false => write!(f, "[{mutability},{extension}]")?,
```

Comment:
The point here is to only print the things that are not default; seems like the commented out code still does that?